en_US/security/authz/file.md

```diff
@@ -34,64 +43,42 @@ The rules file is described in Erlang syntax:
+    * `{username, "dashboard"}` — clients with username `dashboard`.
+    * `{username, {re, "^dash"}}` — clients with username matching the [regular expression](https://www.erlang.org/doc/man/re.html#regexp_syntax) `^dash`.
+    * `{user, ...}` — the same as `{username, ...}`.
+    * `{clientid, "dashboard"}` — clients with clientid `dashboard`.
+    * `{clientid, {re, "^dash"}}` — clients with clientid matching the [regular expression](https://www.erlang.org/doc/man/re.html#regexp_syntax) `^dash`.
+    * `{client, ...}` — the same as `{clientid, ...}`.
+    * `{ipaddr, "127.0.0.1"}` — clients connecting from IP address `127.0.0.1`. Netmasks are allowed. If EMQX is behind a TCP proxy, `proxy_protocol` should be enabled for the client's MQTT listener.
+    * `{ipaddrs, ["127.0.0.1", ..., ]}` — clients connecting from one of the specified IP addresses `127.0.0.1, ..., `. Netmasks are allowed.
+    * `all` — any clients.
+    * `{'and', First, Second}` — clients satisfying both `First` and `Second` specifications.
+    * `{'or', First, Second}` — clients satisfying either of `First` and `Second` specifications.
+- The third position of the tuple indicates the operation for which the rule is applicable.
+    * `publish` — the rule applies to publish operations.
+    * `subscribe` — the rule applies to subscribe operations.
+    * `all` — the rule applies to both publish and subscribe operations.
+- The fourth position of the tuple specifies the topics to which the rule applies. The topics are specified with a list op _patterns_. The following patterns are available:
+    * A string value, like `"$SYS/#"`. It is a standard topic filter allowing wildcards. Topic filters match topics according to the [MQTT specification rules](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/errata01/os/mqtt-v3.1.1-errata01-os-complete.html#_Toc442180920). For example, `$SYS/#` matches topics `$SYS/foo`, `$SYS/foo/bar` for publish and topics
+    `$SYS/foo`, `$SYS/foo/#`, and `$SYS/#` for subscribe. Topic [placeholders](./authz.md#topic-placeholders) are
+    also available.
+    * An `eq` tuple, like `{eq, "foo/#"}`. It indicates full equivalence of topic characters. This pattern matches exactly `foo/#` topic for all operations. Wildcards or placeholders are not taken into account, i.e., topic `foo/bar` is not matched.
+
+Additionally, there are two special rules:
+    - `{allow, all}` — allow all operations.
+    - `{deny, all}` — deny all operations.
+
+These rules are usually used as default at the end of the configuration.
 
 ::: tip
-Only a few simple and general rules is contained in acl.conf that make it a system-based ACL principle. If you need to support complex, large amounts of ACL content, you should implement it in an authentication plugin.
-
+File-based authorization is most suitable for a small number of general rules. If you have many client-specific ACL rules
```

Comment:
move this tip to the top

Reply:
Being on top, this tip now duplicates the similar paragraph, so I removed the tip.